src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -281,17 +281,25 @@ private string GetReportSummary(MigrationReport migrationReport)
             StringBuilder sb = new StringBuilder();
 
             sb.AppendLine("Summary");
-            sb.AppendLine($"Total Projects: {migrationReport.MigratedProjectsCount}");
-            sb.AppendLine($"Succeeded Projects: {migrationReport.SucceededProjectsCount}");
-            sb.AppendLine($"Failed Projects: {migrationReport.FailedProjectsCount}");
+            sb.AppendLine(
```

Comment:
We should add a StringBuilder.AppendLineFormat [naming?] extension method. This pattern is too common in our repo these days... not this PR, of course :)